### DISCUSSION TITLE
Import via CLI and also uploading ios photos

### DISCUSSION THREAD

**Discussion by zorkol** [immich-app/immich#3288] [General]
I did a large import via CLI (30k photos) and they work fine however apple photos on the iphone immich is also uploading 30k photos (the same) and they are appearing as duplicates. What am i doing wrong

**Comment by nicoska84** [comment]
Hi,
how you did the CLI import of the iOS photo? Did you download everything into your Mac and then uploaded to immich server?

**Comment by zorkol** [comment]
Yes I did this but I used a PC

**Reply by nicoska84** [reply]
So, the problem about iOS app trying to upload the same photos still there?

**Comment by zorkol** [comment]
To solve the problem I removed all pictures from immich onthe server and
let the immich app upload everything. It was painstaking slow with 30k
pictures. Now it's working fine though (except the app hanging for 5 sec at
start for checking uploads but  that's a different issue). So I cannot tell
you if any update solved the problem.

**Comment by chris1284** [comment]
i plan to import icloud pics from pc too. i do not have installed the app yet. will i, today, get the same behavior (duplicate import from app and pc?). i know i can run the immich-go duplicate but it would be better that the app do not import already imported pics again. or is the best practise the slower import from iphone with app (70gb)?
